src/libhirte/common/paths.h.in

```diff
@@ -0,0 +1,9 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#pragma once
+
+/* Standard configuration file locations
+ */
+#define CFG_ETC_HIRTE_CONF "@SYSCONFDIR@/hirte/hirte.conf"
```

Comment:
This is a bit weird way of doing this. The typical way is to make SYSCONFDIR a c macro from meson, which will eventually be passed via a -DSYSCONF=\"foo\" compiler option, and then you use it from the C code using string merging like this:

#define CFG_ETC_HIRTE_CONF SYSCONFDIR "/hirte/hirte.conf"